### ISSUE TITLE
Display's IPython's quit/exit instead of quitting with %pdb IPython magic

### ISSUE THREAD

**Issue by alanhdu** [pdbpp/pdbpp#76]
With `pdbpp 0.9`, if you use the `%pdb` magic in an IPython shell, `exit` and `quit` no longer actually exit the pdb session:

```
In [1]: %pdb
Automatic pdb calling has been turned ON

In [2]: raise Exception
---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
<ipython-input-2-2aee0157c87b> in <module>()
----> 1 raise Exception
Exception: 
> <ipython-input-2-2aee0157c87b>(1)<module>()
----> 1 raise Exception

ipdb> quit
<IPython.core.autocall.ExitAutocall object at 0x7fb880b695c0>
ipdb> exit
<IPython.core.autocall.ExitAutocall object at 0x7fb880b695c0>
```

This is particularly irritating in the Jupyter Notebook, because `CTRL-D` to exit pdb does not work like it does in the IPython shell.

This is with pdbpp 0.9.2, Python 3.6, and IPython 6. `exit` and `quit` work correctly with pdbpp 0.8 and 0.8.1 though.

**Comment by alanhdu** [comment]
So it looks like this issue has actually been solved in the latest master. Would it be possible to cut a new release with the fix?

**Comment by blueyed** [comment]
Thanks for the update.

Fixes in master that would be relevant: 0.9.2...b7e789b.
Only 61d0313 seems relevant, and got a better fix by now.

Please consider testing this with current master / ensure that it was really fixed for you with some commit.
It appears to be the same as with your issue though for me, i.e. `quit` and `exit` get displayed, not executed as command.

The workaround is to use just "q" however.
